synapse/lib/storm.py

```diff
@@ -3971,10 +3973,14 @@ async def diffgenr():
                             await runt.printf(f'{nodeiden} {form} = {valurepr}')
                         else:
                             delnode = True
-                            protonode = await editor.addNode(form, valu[0])
+                            if (protonode := await editor.addNode(form, valu[0])) is None:
+                                await asyncio.sleep(0)
+                                continue
```

Comment:
In the `--wipe` case, should we be throwing an exception instead of skipping? I worry about data being lifted and failing to merge [ for whatever reason ] and then being deleted without user intervention.

Reply:
We explicitly set strict=False, but maybe we should set it to True in the `--wipe`.  Probably worth a quick goback.

Reply:
Per disucssion, leave the behavior as is and update the `--wipe` documentation.